AK/StringBase.h

```diff
@@ -70,6 +70,8 @@ class StringBase {
 
     void did_create_fly_string(Badge<FlyString>) const;
 
+    [[nodiscard]] ALWAYS_INLINE FlatPtr raw() const { return bit_cast<FlatPtr>(m_data); }
```

Comment:
Feels like this should have a `Badge<FlyString>`, otherwise it's exposed on all `String`s